### PR TITLE
Update README to meet proposed documentation criteria

### PR DESCRIPTION
**Pull request by zuren**
The criteria are:

- [x] Business context
- [x] Stack
- [x] Installation
- [x] Development practices
- [x] Release process
- [x] Deployment pipeline
- [ ] Common problems
- [x] Useful contacts
- [ ] API endpoints (probably to come automatically)

Please feedback on the criteria and whether they're met well.
Conscious some sections are lacking.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
+
+## Contributing
 
-## Development
+### Setup
 
-To serve the application, run the following and visit [http://localhost:3000](http://localhost:3000)
+1. Install [Docker][docker-download].
+2. Clone this repository.
+3. Run `make build`.
```
Comment (lukemorton) [inline README.md]: We might want to consider traditional Microsoft users as being an audience of this content. `make` should probably be a prerequisite along with `Docker`.

File: README.md [hunk 2]
```diff
@@ -45,4 +72,25 @@ Universal Housing configuration is given through environment variables, for exam
 - UH_DATABASE_HOST=universal_housing
 - UH_DATABASE_PORT=1433
 
-When developing locally, the docker compose configuration assumes that the [Universal Housing simulator](https://github.com/LBHackney-IT/lbh-universal-housing-simulator) is checked out in the same directory as this.
+When developing locally, the Docker Compose configuration assumes you have access to ECR, where the latest [Universal Housing simulator][github-uh-simulator] Docker image is hosted.
```
Comment (lukemorton) [inline README.md]: This might need a little more explanation.

File: README.md [hunk 3]
```diff
@@ -20,22 +36,33 @@ To run tests:
...
-## Connection to Universal Housing
+### Release
+
+We use a pull request workflow, where changes are made on a branch and approved by one or more other maintainers before the developer can merge into `master`.
+
+Then we have an automated four step deployment process, which runs in CircleCI.
+
+1. Automated tests (RSpec) and linting (Rubocop) are run to ensure the release is of good quality.
+2. The app is deployed to staging automatically, where we check our latest changes work well.
+3. We manually confirm a production deployment once we're happy with our changes in staging.
```
Comment (StevenLeighton21) [inline README.md]: Might clarify that this manual confirmation is part of the Circle CI workflow, if this documentation assumes users are unfamiliar with the stack

File: README.md (hunk 2 above)
Comment (cob16) [reply README.md]: This can be left as is due to #57

File: README.md (hunk 1 above)
Comment (zuren) [reply README.md]: Good point, should do something about this. http://gnuwin32.sourceforge.net/packages/make.htm might be enough?

Would prefer a native cross platform alternative, but can't think of a good one.

File: README.md (hunk 3 above)
Comment (zuren) [reply README.md]: Good idea.

File: README.md (hunk 2 above)
Comment (zuren) [reply README.md]: Sure.

**Pull request merged**